### PR TITLE
InfluxDB to BQ ETL

### PR DESCRIPTION
**Pull request by alekhyamoz**
Relates to https://mozilla-hub.atlassian.net/browse/RS-683. The job imports data from InfluxDB to BQ
TODO - schedule the job via Airflow to run daily

### CODE REVIEW

**Review by lelilia — APPROVED** [review]
lgtm

File: jobs/influxdb-to-bigquery/influxdb_to_bigquery/main.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+@click.command()
+@click.option("--bq_project_id", help="GCP project id", required=True)
+@click.option("--bq_dataset_id", help="GCP dataset id", required=True)
+@click.option(
+    "--bq_table_id",
+    help="GCP table id",
+    required=True,
+)
+@click.option("--influxdb_measurement", help="Influx measurement", required=True)
```
Comment (akkomar) [inline jobs/influxdb-to-bigquery/influxdb_to_bigquery/main.py:70]:
```suggestion
@click.option("--influxdb_measurement", help="Influx measurement to fetch", required=True)
```

**Review by akkomar — CHANGES_REQUESTED** [review]
Overall LGTM but I left some comments. Above all the README template should be filled :)

File: jobs/influxdb-to-bigquery/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Python Template Job
```
[the remark this answers is not in the document]
Comment (alekhyamoz) [reply jobs/influxdb-to-bigquery/README.md:1]: Will do, forgot to add

**Review by akkomar — APPROVED** [review]

**Pull request merged**